app/frontend/src/components/forms/ShareForm.vue

```diff
@@ -80,14 +88,13 @@ export default {
     formId: {
       type: String,
       required: true,
-    }
+    },
   },
   data() {
     return {
       dialog: false,
-      qrShow: false,
-      qrSize: 400,
-      ticksLabels: ['Small', 'Medium', 'Large'],
+      qrLevel: 'M',
+      qrSize: 900,
```

Comment:
This value should be used on Line 45 instead of hard coded in.

Reply:
Oh yeah was playing around with size and forgot that. I thought Vue warned when unused Data fields existed...